code/game/gamemodes/changeling/powers/headcrab.dm

```diff
@@ -30,9 +30,9 @@
 	for(var/mob/living/silicon/S in range(2,user))
 		to_chat(S, "<span class='userdanger'>Your sensors are disabled by a shower of blood!</span>")
 		S.Weaken(3)
-	var/turf = get_turf(user)
+	var/new_location = user.loc
```

Comment:
This is really dangerous, to say the least. There's areas a changeling could end up, die in, or need to burst in that are NOT compatible, at all, with `user.loc`.

Reply:
I see. What do you recommend instead? drop_location() sent the thing to nullspace. Simply used what Ansari suggested to use and it worked. Super new to coding, didn't know this was the case.